internal/monitors/logstash/logstash/metadata.yaml

```diff
@@ -0,0 +1,427 @@
+      default: false
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.plugins.codecs.encode.writes_in:
+      description: Number of incoming encode events to codecs
+      default: false
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.filters.events.duration_in_millis:
+      description: Duration of events in filter plugins
+      default: true
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.plugins.filters.events.in:
+      description: Number of incoming events to filters
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.filters.events.out:
+      description: Number of outgoing events from filters
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.inputs.events.out:
+      description: Number of outgoing events from inputs
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.inputs.events.queue_push_duration_in_millis:
+      description: Queue push duration of events in input plugins
+      default: false
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.plugins.outputs.events.duration_in_millis:
+      description: Duration of events in output plugins
+      default: true
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.plugins.outputs.events.in:
+      description: Number of incoming events to output plugins
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.outputs.events.out:
+      description: Number of outgoing events from output plugins
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.queue.events_count:
+      description: Number of events in queue
+      default: false
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.queue.max_queue_size_in_bytes:
+      description: Max queue size in pipelines
+      default: false
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.queue.queue_size_in_bytes:
+      description: Queue size in pipelines
+      default: false
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.reloads.failures:
+      description: Number of failed reloads
+      default: false
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.reloads.successes:
+      description: Number of successful reloads
+      default: false
+      type: cumulative
+      group: pipeline
+    node.stats.events.events.duration_in_millis:
+      description: Duration of events
+      default: true
+      type: gauge
+      group: events
+    node.stats.events.events.filtered:
+      description: Number of filtered events
+      default: true
+      type: cumulative
+      group: events
+    node.stats.events.events.in:
+      description: Number of incoming events
```

Comment:
Rephrase to make it clear that this means the total number of events input to the system, not currently considered "incoming".